systemtest/src/test/java/io/strimzi/systemtest/watcher/AbstractNamespaceST.java

```diff
@@ -4,67 +4,326 @@
+     *
+     * @usecase
+     *  - namespaces
+     *  - topic-operator-watcher
+     */
+    @IsolatedTest
+    @KRaftNotSupported("TopicOperator is not supported by KRaft mode and is used in this test case")
+    final void testTopicOperatorWatchingOtherNamespace(ExtensionContext extensionContext) {
+
+        LOGGER.info("Topic Operator in Kafka: {}/{} watches KafkaTopics in (different) Namespace: {}", MAIN_TEST_NAMESPACE, PRIMARY_KAFKA_NAME, PRIMARY_KAFKA_WATCHED_NAMESPACE);
+
+        String topicName = mapWithTestTopics.get(extensionContext.getDisplayName());
+        LOGGER.info("Verifying that KafkaTopic: {}/{} does not exist before test", PRIMARY_KAFKA_WATCHED_NAMESPACE, topicName);
+        List<String> topics = KafkaCmdClient.listTopicsUsingPodCli(MAIN_TEST_NAMESPACE, scraperPodName, KafkaResources.plainBootstrapAddress(PRIMARY_KAFKA_NAME));
+        assertThat(topics, not(hasItems(topicName)));
+
+        LOGGER.info("Verifying that KafkaTopic: {}/{} is watched by TO by asserting its existence", PRIMARY_KAFKA_WATCHED_NAMESPACE, topicName);
+        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(PRIMARY_KAFKA_NAME, topicName, PRIMARY_KAFKA_WATCHED_NAMESPACE).build());
+        topics = KafkaCmdClient.listTopicsUsingPodCli(MAIN_TEST_NAMESPACE, scraperPodName, KafkaResources.plainBootstrapAddress(PRIMARY_KAFKA_NAME));
+        assertThat(topics, hasItems(topicName));
+    }
+
+    // TODO would remove as redundant in favour of testUserInDifferentNamespace -> testUserOperatorWithUserWatchedInSeparatedNamespace
+    /**
+     * @description This test case verifies that User Operator configured to watch other namespace than the one it is deployed in still watches and acts upon
+     * custom resources correctly.
+     *
+     * @steps
+     *  1. - As part of setup Kafka Cluster is deployed in main namespace, with the User Operator configured to watch other namespace.
+     *     - Kafka and its components are deployed and ready.
+     *  2. - KafkaUser custom resource is created in namespace watched by Topic Operator.
+     *     - Topic Operator acts upon KafkaUser custom resource which is transitioned into ready state.
+     *
+     * @usecase
+     *  - namespaces
+     *  - user-operator-watcher
+     */
+    @IsolatedTest
+    final void testUOWatchingOtherNamespace(ExtensionContext extensionContext) {
+        final TestStorage testStorage = new TestStorage(extensionContext);
+
+        LOGGER.info("Creating KafkaUser: {}/{} in namespace different from on Kafka Cluster or Cluster Operator are located in", PRIMARY_KAFKA_WATCHED_NAMESPACE, testStorage.getUsername());
+        resourceManager.createResource(extensionContext, KafkaUserTemplates.tlsUser(PRIMARY_KAFKA_WATCHED_NAMESPACE, PRIMARY_KAFKA_NAME, testStorage.getUsername()).build());
+        LOGGER.info("KafkaUser: {}/{} was successfully created", PRIMARY_KAFKA_WATCHED_NAMESPACE, testStorage.getUsername());
+    }
```

Comment:
I kept this test but i i think we could actually remove it as it is pretty much covered by another test (as mentioned in this TODO)

Reply:
Did you mean `testUserInNamespaceDifferentFromUserOperator` test? I cannot find the test you are mentioning in the TODO.
If so, I guess you can remove it

Reply:
thanks will remove.